interview-questions.md

```diff
@@ -14,44 +14,44 @@ Researched answer:
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: Git is a program that can access files and save files. Github is a website where you can send and pull files using the program git.
 
-Researched answer:
+Researched answer: I found that Git is a version control system that lets user manage projects, source code history. Git makes use of repositories and track each project independently. Github on the other hand is like a cloud where you can store the repositories. The website serves as a host that allows user to access the projects anywhere and lets users share their projects to another user.
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: The JavaScript operators that will return a Boolean value true or false are loose equality (==), strict equality (===), and greater than (>) and less than (<) symbols.
 
-Researched answer:
+Researched answer: I found that there are different JavaScript Operators that can return a Boolean value. First is Equality Operators which includes the loose equality using ==, and strict equality using ===. Second is the Relational Operators that uses <, >, <=, >=. Next, is the Logical Operators that uses && which means AND, || which means OR, and !== which means OPPOSITE.
 
 3. What is an index? What is the difference between index and value?
 
-Your answer:
+Your answer: An index is the address or the position of a value. The value is the data of a variable.
 
-Researched answer:
+Researched answer: I found that index is the location or address of an element in the array. The are numbered that start from 0 and increases by 1 per position. Value on the other hand is the element in the array.
 
 4. What is iteration?
 
-Your answer:
+Your answer: Iteration means to manipulate the values to get an output.
 
-Researched answer:
+Researched answer: I found that interation in programming are instructions using code in order to achieve a result. In JavaScript the use of iterators defines a sequence to potentially return a value.
```

Comment:
These instructions allow the code to repeat until a condition is met for it to stop.